contracts/utils/GovernorTimelockControl.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockControl.sol)
+
+pragma solidity ^0.8.30;
```

Comment:
This is a full copy of the original 4.8.3 [GovernorTimelockControl](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.8.3/contracts/governance/extensions/GovernorTimelockControl.sol), with all the added parts highlighted below